package.json

```diff
@@ -136,7 +136,8 @@
   },
   "dependencies": {
     "@types/ssh2": "0.5.46",
+    "cspell": "5.4.0",
     "artillery": "1.7.2",
-    "cspell": "5.4.0"
+    "web3": "1.3.6"
```

Comment:
@travis-payne The project root package.json is meant to only hold the dependencies needed to perform the building of the child packages where the code is => please remove this dependency declaration from.